### PR TITLE
feat(themes): theming example with angular material

### PR DESCRIPTION
**Pull request by SisIvanova**
Related to: IgniteUI/igniteui-docfx/issues/2726

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+
+$my-app-theme: mat-light-theme((
+  color: (
+    primary: $my-app-primary,
+    accent: $my-app-accent,
+  )
+));
+
+@include angular-material-theme($my-app-theme);
+
```
Comment (simeonoff) [inline src/styles.scss:15]: Why do we need this in the global styles?

Comment (SisIvanova) [reply src/styles.scss:15]: If we remove the first two lines, the layout of the Material dropdown menus will break. I will delete the rest of the styles.

Comment (simeonoff) [reply src/styles.scss]: Why? Aren't you already including the core angular material styles as part of the sample?

File: src/app/theming/angular/angular-sample.component.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { OverlayContainer } from '@angular/cdk/overlay';
+
+@Component({
+    selector: 'app-angular-sample',
+    styleUrls: ['./angular-sample.component.scss'],
+    templateUrl: './angular-sample.component.html'
+})
+
+export class AngularComponent implements OnInit {
```
Comment (simeonoff) [inline src/app/theming/angular/angular-sample.component.ts:12]:
```suggestion
export class AngularMaterialComponent implements OnInit {
```

**Review by simeonoff — CHANGES_REQUESTED** [review]

**Review by simeonoff — APPROVED** [review]

**Pull request merged**